logd/src/tests/rpc_forwarder/rpc_forwarder_test.cpp

```diff
@@ -31,31 +31,28 @@ decode_log_request(FRT_Values& src, ProtoConverter::ProtoLogRequest& dst)
 std::string garbage("garbage");
 
 struct RpcServer : public FRT_Invokable {
-    FRT_Supervisor supervisor;
+    fnet::frt::StandaloneFRT server;
     int request_count;
     std::vector<std::string> messages;
     bool reply_with_error;
     bool reply_with_proto_response;
 
 public:
     RpcServer()
-        : supervisor(),
+        : server(),
           request_count(0),
           messages(),
           reply_with_error(false),
           reply_with_proto_response(true)
     {
-        supervisor.Listen(0);
-        FRT_ReflectionBuilder builder(&supervisor);
+        server.supervisor().Listen(0);
```

Comment:
listen needs to be moved past reflection building now that start is implicit.

Reply:
Fixed